src/Umbraco.Configuration/Models/WebRoutingSettings.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Microsoft.Extensions.Configuration;
+using Umbraco.Core.Configuration.UmbracoSettings;
+using Umbraco.Core.Models.PublishedContent;
+
+namespace Umbraco.Configuration.Models
+{
+    internal class WebRoutingSettings : IWebRoutingSettings
+    {
+        private readonly IConfiguration _configuration;
+
+        public WebRoutingSettings(IConfiguration configuration)
+        {
+            _configuration = configuration;
+        }
+
+        public bool TrySkipIisCustomErrors =>
```

Comment:
Don't think this is going to make a lot of sense? ... maybe something to revisit later on

Reply:
Agree.. This in just to have the same interfaces as before, for now.. Make sense to clean up when we don't have to care about the .NET Framework executable